stacks/dovetail.prx.org.yml

```diff
@@ -377,6 +377,13 @@ Resources:
                   # - !Sub ${AWS::StackName}-DovetailService TODO
       ManagedPolicyArns:
         - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+      Tags:
+        - Key: Project
+          Value: Infrastructure
```

Comment:
Not "Dovetail"?

Reply:
Nevermind - i see this is the task count rollover (which may no longer be necessary anyways).